### PR TITLE
update entity-management to 1.0.20, bluepy to 0.14.5

### PR DESCRIPTION
**Pull request by genric**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-entity-management/package.py [hunk 1]
```diff
@@ -12,11 +12,8 @@ class PyEntityManagement(PythonPackage):
     homepage = 'https://bbpcode.epfl.ch/code/#/admin/projects/nse/entity-management'
     git      = 'ssh://bbpcode.epfl.ch/nse/entity-management'
 
+    version('1.0.20', tag='entity-management-v1.0.20')
     version('1.0.18', tag='entity-management-v1.0.18')
-    version('1.0.16', tag='entity-management-v1.0.16')
```
Comment (wizmer) [inline var/spack/repos/builtin/packages/py-entity-management/package.py:16]: No reason to remove the old tags, no ?

Comment (genric) [reply var/spack/repos/builtin/packages/py-entity-management/package.py:16]: I hope they will be available in the HPC archives, but for the current set of modules they keep anyways only the last two.

**Review by wizmer — APPROVED** [review]

**Review by tomdele — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-entity-management/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/py-entity-management/package.py:16]: Depends on deployment frequency…

Comment (genric) [reply var/spack/repos/builtin/packages/py-entity-management/package.py:16]: Ah, ok, that was my other guess. Thanks @matz-e!

**Pull request merged**